### ISSUE TITLE
Webcam Images & Videos on Desktop/Web chat (Use webcam to create images and videos to send in chat)

### ISSUE THREAD

**Issue by trymeouteh** [dino/dino#1006]
Please allow for the desktop client to easily send images and video taken from the webcam in the desktop client in the chat. Just like how you can use your camera on your phone to take pictures or videos and send it in the chat easily but in the Dino client.

**Comment by Echolon** [comment]
Hmm, duplicate of #154?

**Comment by bwildenhain** [comment]
> Hmm, duplicate of #154?

No, this is about asynchronous communication (like e-mails), while #154 is about synchronous communication (like phone calls).
